src/app/pages/currency-converter/converter.controller.js

```diff
@@ -67,6 +68,9 @@ export default class ConverterController {
 
         this.syncDataService.addDealToFirebase(userDeal);
         this.syncDataService.getAllFromFirebase(); // it's just for check. It show all deals in console
+        this.tradeValue !== null ? this.toastr.success('Congrats! Successful deal!') : this.toastr.error('Ooops! How much trade?'); 
```

Comment:
You may write:
this.tradeValue ? this.toastr.success('Congrats! Successful deal!') : this.toastr.error('Ooops! How much trade?');